app/database/QueryBuilder.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
prefer kebab-case.php for filenames to avoid case sensitivity pitfalls across development operating systems

Reply:
Gotcha, I will make that change